addon/services/emoji.js

```diff
@@ -107,71 +98,31 @@ export default Service.extend({
-
-  people__tone_2:   filterBy('emoji__tone_2', 'category', 'people'),
-  nature__tone_2:   filterBy('emoji__tone_2', 'category', 'nature'),
-  food__tone_2:     filterBy('emoji__tone_2', 'category', 'food'),
-  activity__tone_2: filterBy('emoji__tone_2', 'category', 'activity'),
-  travel__tone_2:   filterBy('emoji__tone_2', 'category', 'travel'),
-  objects__tone_2:  filterBy('emoji__tone_2', 'category', 'objects'),
-  symbols__tone_2:  filterBy('emoji__tone_2', 'category', 'symbols'),
-  flags__tone_2:    filterBy('emoji__tone_2', 'category', 'flags'),
-  regional__tone_2: filterBy('emoji__tone_2', 'category', 'regional'),
-  modifier__tone_2: filterBy('emoji__tone_2', 'category', 'modifier'),
-
-  people__tone_3:   filterBy('emoji__tone_3', 'category', 'people'),
-  nature__tone_3:   filterBy('emoji__tone_3', 'category', 'nature'),
-  food__tone_3:     filterBy('emoji__tone_3', 'category', 'food'),
-  activity__tone_3: filterBy('emoji__tone_3', 'category', 'activity'),
-  travel__tone_3:   filterBy('emoji__tone_3', 'category', 'travel'),
-  objects__tone_3:  filterBy('emoji__tone_3', 'category', 'objects'),
-  symbols__tone_3:  filterBy('emoji__tone_3', 'category', 'symbols'),
-  flags__tone_3:    filterBy('emoji__tone_3', 'category', 'flags'),
-  regional__tone_3: filterBy('emoji__tone_3', 'category', 'regional'),
-  modifier__tone_3: filterBy('emoji__tone_3', 'category', 'modifier'),
-
-  people__tone_4:   filterBy('emoji__tone_4', 'category', 'people'),
-  nature__tone_4:   filterBy('emoji__tone_4', 'category', 'nature'),
-  food__tone_4:     filterBy('emoji__tone_4', 'category', 'food'),
-  activity__tone_4: filterBy('emoji__tone_4', 'category', 'activity'),
-  travel__tone_4:   filterBy('emoji__tone_4', 'category', 'travel'),
-  objects__tone_4:  filterBy('emoji__tone_4', 'category', 'objects'),
-  symbols__tone_4:  filterBy('emoji__tone_4', 'category', 'symbols'),
-  flags__tone_4:    filterBy('emoji__tone_4', 'category', 'flags'),
-  regional__tone_4: filterBy('emoji__tone_4', 'category', 'regional'),
-  modifier__tone_4: filterBy('emoji__tone_4', 'category', 'modifier'),
-
-  people__tone_5:   filterBy('emoji__tone_5', 'category', 'people'),
-  nature__tone_5:   filterBy('emoji__tone_5', 'category', 'nature'),
-  food__tone_5:     filterBy('emoji__tone_5', 'category', 'food'),
-  activity__tone_5: filterBy('emoji__tone_5', 'category', 'activity'),
-  travel__tone_5:   filterBy('emoji__tone_5', 'category', 'travel'),
-  objects__tone_5:  filterBy('emoji__tone_5', 'category', 'objects'),
-  symbols__tone_5:  filterBy('emoji__tone_5', 'category', 'symbols'),
-  flags__tone_5:    filterBy('emoji__tone_5', 'category', 'flags'),
-  regional__tone_5: filterBy('emoji__tone_5', 'category', 'regional'),
-  modifier__tone_5: filterBy('emoji__tone_5', 'category', 'modifier'),
+  init() {
+    this._defineEmojiCategoryProperties();
+    this._defineEmojiToneProperties();
+  },
```

Comment:
Nope, services don't require it:
> "By default, does nothing unless it is overridden during class definition."
http://emberjs.com/api/classes/Ember.Service.html#method_init